Source/Core/Proj4Projection.js

```diff
@@ -0,0 +1,196 @@
+define([
+        './Cartesian3',
+        './Cartographic',
+        './Math',
+        './Check',
+        './defaultValue',
+        './defined',
+        './defineProperties',
+        './Ellipsoid',
+        './oneTimeWarning',
+        './Rectangle',
+        '../ThirdParty/proj4-2.5.0'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        CesiumMath,
+        Check,
+        defaultValue,
+        defined,
+        defineProperties,
+        Ellipsoid,
+        oneTimeWarning,
+        Rectangle,
+        proj4) {
+    'use strict';
+
+    // proj4 may crash with absolute extents
+    var smallerThanMax = Rectangle.fromDegrees(-180.0 + CesiumMath.EPSILON7, -90.0 + CesiumMath.EPSILON7, 180 - CesiumMath.EPSILON7, 90 - CesiumMath.EPSILON7);
```

Comment:
How were these epsilons chosen?  Would `Number.MIN_VALUE` be a better choice if the goal is to be the smallest number that is not the max?